vendor.conf

```diff
@@ -37,7 +37,7 @@ github.com/mitchellh/hashstructure 2bca23e0e452137f789efbc8610126fd8b94f73b
 #get libnetwork packages
 
 # When updating, also update LIBNETWORK_COMMIT in hack/dockerfile/install/proxy.installer accordingly
-github.com/docker/libnetwork 49627167f0585504fd78ed8827529aec57a9618d
+github.com/docker/libnetwork 1f28166bb386cf9223d2d00a28382b0e474be314
```

Comment:
Can you move the vendoring changes to a separate commit? (this makes it easier to backport if that turns out to be needed)